modules/developers-guide/pages/metrics.adoc

```diff
@@ -0,0 +1,36 @@
+= Using metrics with Stargate
+
+Stargate is a data gateway deployed between client applications and a database.
+Metrics can be monitored using Prometheus and Grafana. A
+https://github.com/stargate/docker-images/[`docker-compose`
+script] is available to run Apache Cassandra^(TM)^, Stargate, Prometheus, and Grafana
+together to explore the metrics available.
+
+// tag::prereqsList[]
+include::partial$prereqs.adoc[]
+// end::prereqsList[]
+
+// tag::getDockerImage[]
+include::partial$docker_compose.adoc[]
+// end::getDockerImage[]
+
+== Access Prometheus
+Once all the images are running, you can access Prometheus in the browser at
+http://localhost:9090/[http://localhost:9090/].
+
+In order to access the Stargate metrics, you must modify the `/etc/prometheus/prometheus.yml`
+file on the Prometheus node to change the target from `localhost:9090` (the Prometheus node)
+to `0.0.0.0:8085` (`8085` is the Stargate node's metrics port).
+Restart the Prometheus node and the Stargate metrics will be available.
+The Stargate metrics are grouped by API, and can accessed as any other metrics are
+accessed in http://prometheus.io[Prometheus].
+
+== Access Grafana
+Once all the images are running, you can access Grafana in the browser at
+http://localhost:3000/[http://localhost:3000/].
+You'll need to login with the default username and password, `admin/admin`.
+Add Prometheus as a data source. You'll need to edit the HTTP URL field to add
```

Comment:
> Add Prometheus as a data source

How? Maybe "Click data source icon on the left hand side and then fill out the form like below" and have a screen grab? Should be mostly defaults I believe but for those not familiar with Grafana it would be useful.